packages/dht/src/connection/websocket/WebsocketServer.ts

```diff
@@ -139,7 +131,10 @@ export class WebsocketServer extends EventEmitter<ConnectionSourceEvents> {
         this.abortController.abort()
         this.removeAllListeners()
         return new Promise((resolve, _reject) => {
-            this.wsServer?.shutDown()
+            this.wsServer!.close()
```

Comment:
It would maybe make sense to close the connections (i.e. clients) first and then the server? If the behavior of closed server is not well defined.

Reply:
The WS server should always be closed first. This ensures that new connections are no longer accepted